libs/libvtrutil/src/vtr_bimap.h

```diff
@@ -92,6 +109,37 @@ class bimap {
     ///@brief Return true if the specified value exists
     bool contains(const V value) const { return find(value) != inverse_end(); }
 
+  public: //Constructors/Deconstructor
+    ///@brief default constructor required by compiler
+    bimap() = default;
+
+    ///@brief default deconstructor required by compiler
+    ~bimap() = default;
+
+    ///@brief copy constructor
+    bimap(const bimap<K, V, Map, InvMap>& other)
+        : map_(other.map_)
+        , inverse_map_(other.inverse_map_) {}
+
+    ///@brief move constructor
+    bimap(const bimap<K, V, Map, InvMap>&& other) {
+        map_ = std::move(other.map_);
+        inverse_map_ = std::move(other.inverse_map_);
+        this->clear();
+    }
+
+    ///@brief construct the bimap using initializer list with value_type
+    bimap(std::initializer_list<value_type> il) {
+        for (auto rec : il)
```

Comment:
make it a `const auto &rec`, otherwise it copies the pair in each iteration (also: in the reverse case).